inc/sailag.h

```diff
@@ -176,6 +176,33 @@ typedef enum _sai_lag_attr_t
      */
     SAI_LAG_ATTR_LABEL,
 
+    /**
+     * @brief Adaptive routing and switching object for LAG.
+     *
+     * @type sai_object_id_t
+     * @flags CREATE_AND_SET
+     * @objects SAI_OBJECT_TYPE_ARS
+     * @allownull true
+     * @default SAI_NULL_OBJECT_ID
+     */
+    SAI_LAG_ATTR_ARS_OBJECT_ID,
+
+    /**
+     * @brief Number of packets dropped by ARS mechanism
```

Comment:
please clarify when packets can be dropped by ARS

Reply:
Most common usecase when packets may get dropped is when path selection happens because of link going down. Even though reassignment happens in HW there is a possibility of packets getting dropped.